src/screens/IndexScreen.js

```diff
@@ -49,19 +32,55 @@ export default class IndexScreen extends React.Component {
     const { navigation } = this.props;
 
     return (
-      <View style={styles.container}>
-        <Text>Index Screen</Text>
-        {items.map((item) => (
-          <Button
-            key={`bla${item.itemId}`}
-            title={`Got to Detail #${item.itemId}`}
-            // on press navigate to Detail route (DetailScreen) with the following params,
-            // that we use in that screen
-            onPress={() => navigation.navigate('Detail', item)}
-            color={colors.primary}
-          />
-        ))}
-      </View>
+      <Query query={GET_CACHE_ITEMS} fetchPolicy="cache-only">
+        {({ data, client }) => {
+          const { cacheItems } = data;
+
+          return (
+            <View style={styles.container}>
+              <Text>Index Screen</Text>
+              {!!cacheItems &&
+                cacheItems.map((item) => (
+                  <Button
+                    key={`bla${item.itemId}`}
```

Comment:
To do : bla needs to be changed with a appropriate word

Reply:
thx, changed it to `key`.